api/v1/db/oracledb/game-dao.js

```diff
@@ -124,27 +147,32 @@ const getGamesByMemberId = async (id, query) => {
 };
 
 /**
- * @summary Check the if the game with the id exists.
+ * @summary Check if a certain member is already in the game by memberId and gameId parameters.
  * @function
- * @param {number} id The id of the game
- * @returns {Promise<boolean>} If the game exists or not.
+ * @param {number} memberId
+ * @param {number} gameId
+ * @returns {boolean} If the member is already in the game.
```

Comment:
Could you change `boolean` to `Promise<boolean>` since this is an async function?